#!/usr/bin/env python
PACKAGE = "ohm_blue_bars"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("Hmax", int_t, 0, "hue max",	130, 0, 179)
gen.add("Hmin", int_t, 0, "hue min", 	90,   0, 179)
gen.add("Smax", int_t, 0, "saturation max",  	255, 0, 255)
gen.add("Smin", int_t, 0, "saturation min",    50,  0, 255)
gen.add("Vmax", int_t, 0, "value max",  255, 0, 255)
gen.add("Vmin", int_t, 0, "value min",  110,  0, 255)
gen.add("sizeA", int_t, 0, "sizeA",     7,   3, 20)
gen.add("sizeB", int_t, 0, "sizeB", 	7,   3, 20)
gen.add("pointA", int_t, 0, "pointA", 	3,	 1, 20)
gen.add("pointB", int_t, 0, "pointB", 	3,   1, 20)
gen.add("thresh_pxl_detect", int_t, 0, "thresh_pxl_detect", 100, 0, 255)
gen.add("threshScore", int_t, 0, "ThreshScore", 60, 0, 255) 

gen.add("intersections", int_t, 0, "Intersections", 110, 0, 300) 



exit(gen.generate(PACKAGE, "ohm_blue_bars", "BlueBarsCfg"))
